tests/utils/test_post_docs_data.py

```diff
@@ -0,0 +1,49 @@
+import os
+import importlib
+import pytest
+import snowflake.connector
+from utils import post_docs_data
+import responses
+
+class TestPostDocsJob():
```

Comment:
@munish7771 , adding some comments regarding the purpose of these tests would be very useful. That way one does not have to jump to the original github PR to get the context (and it might not be always available)